### PR TITLE
Add builder Dao proposals to Governance Fidget 

### PR DESCRIPTION
**Pull request by sktbrd**
- Added a separated files for proposal queries 
- Created a couple of components under community fidgets 
      - ProposalItem and ProposalView for Builder Daos 
- Adapted NounishGovernance to differentiate different query types, currently: 
      - Nouns ETH dao 
      - Base Builder dao

### CODE REVIEW

File: src/common/components/organisms/FidgetSettingsEditor.tsx [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-  FidgetSettings,
-  FidgetProperties,
-  FidgetFieldConfig,
-} from "@/common/fidgets";
-import BackArrowIcon from "../atoms/icons/BackArrow";
-import { FaTrashCan, FaTriangleExclamation } from "react-icons/fa6";
 import { Button } from "@/common/components/atoms/button";
 import {
   Tabs,
+  TabsContent,
```
Comment (hiporox) [inline src/common/components/organisms/FidgetSettingsEditor.tsx]: From what I can tell, this is all the same content, but reordered. Can you clean this up to either make it clear that nothing has changed, or highlight the changes in a real way?

File: src/fidgets/community/nouns-dao/NounishGovernance.tsx [hunk 2]
```diff
@@ -1,14 +1,19 @@
-import React, { useState } from "react";
-import { FidgetArgs, FidgetProperties, FidgetModule } from "@/common/fidgets";
-import type { ProposalData } from "@/fidgets/community/nouns-dao";
-import useGraphqlQuery from "@/common/lib/hooks/useGraphqlQuery";
-import ProposalListView from "@/fidgets/community/nouns-dao/components/ProposalListView";
-import ProposalDetailView from "@/fidgets/community/nouns-dao/components/ProposalDetailView";
 import { CardContent } from "@/common/components/atoms/card";
 import TextInput from "@/common/components/molecules/TextInput";
+import { FidgetArgs, FidgetModule, FidgetProperties } from "@/common/fidgets";
```
Comment (hiporox) [inline src/fidgets/community/nouns-dao/NounishGovernance.tsx]: Once again, you clean this up to make the changes that are being made more obvious. By changing the import order, you obfuscate what changes are being made

File: src/fidgets/community/nouns-dao/components/ProposalDetailView.tsx [hunk 3]
```diff
@@ -1,19 +1,15 @@
...
+import { mergeClasses } from "@/common/lib/utils/mergeClasses";
+import type { ProposalData } from "@/fidgets/community/nouns-dao";
 import moment from "moment";
+import { FaArrowLeft } from "react-icons/fa6";
+import { RiExternalLinkLine } from "react-icons/ri";
 import { useEnsName } from "wagmi";
 import { mainnet } from "wagmi/chains";
-
+import { StatusBadge, estimateBlockTime } from "./proposalListRowItem";
+import React from "react";
```
Comment (hiporox) [inline src/fidgets/community/nouns-dao/components/ProposalDetailView.tsx]: Again, why the re-ordering of imports

File: tsconfig.json [hunk 4]
```diff
@@ -50,4 +50,4 @@
     "node_modules",
     "./node_modules"
   ]
-}
+}
```
Comment (hiporox) [inline tsconfig.json]: No need to remove the newline

File: src/fidgets/community/nouns-dao/components/proposalListRowItem.tsx
Comment (hiporox) [inline src/fidgets/community/nouns-dao/components/proposalListRowItem.tsx:1]: Start file name casing with a capital letter for Component files

File: src/fidgets/community/nouns-dao/components/builderProposalItem.tsx
Comment (hiporox) [inline src/fidgets/community/nouns-dao/components/builderProposalItem.tsx:1]: Start file name casing with a capital letter for Component files

File: src/fidgets/community/nouns-dao/NounishGovernance.tsx [hunk 5]
```diff
@@ -22,6 +27,12 @@ export const nounishGovernanceConfig: FidgetProperties = {
       required: true,
       inputSelector: TextInput,
     },
+    {
+      fieldName: "daoContractAddress",
+      default: "Only for Builder Daos",
+      required: true,
```
Comment (hiporox) [inline src/fidgets/community/nouns-dao/NounishGovernance.tsx]: Is this actually required, or only necessary if other things are true? You don't need to fix this right now, but I am gauging how much conditional validation should be built soon

**Review by hiporox — CHANGES_REQUESTED** [review]
All of the real functionality here looks good to me, but there are a lot of changes here that seem unnecessary. Can you please clean them up?

File: src/common/components/organisms/FidgetSettingsEditor.tsx (hunk 1 above)
Comment (sktbrd) [reply src/common/components/organisms/FidgetSettingsEditor.tsx]: yeah sorry about that, I have this vscode settings that when I save a file it reorder the imports. and as it was my first time exploring the code (with thelaunching rush) I have autosaved some files after that plugin I use reorganize it. ups

File: src/fidgets/community/nouns-dao/NounishGovernance.tsx (hunk 5 above)
Comment (sktbrd) [reply src/fidgets/community/nouns-dao/NounishGovernance.tsx]: its only necessary for builder daos, the eth main Nouns dao does not need that field. I wanted to get the structure of the components done so I can now improve the UX. Glad that worked though , just moon from here!

File: src/fidgets/community/nouns-dao/NounishGovernance.tsx (hunk 2 above)
Comment (sktbrd) [reply src/fidgets/community/nouns-dao/NounishGovernance.tsx]: same as this: https://github.com/Nounspace/nounspace.ts/pull/176/files#r1661673126

File: src/fidgets/community/nouns-dao/components/ProposalDetailView.tsx (hunk 3 above)
Comment (sktbrd) [reply src/fidgets/community/nouns-dao/components/ProposalDetailView.tsx]: same as this: https://github.com/Nounspace/nounspace.ts/pull/176/files#r1661673126

File: src/common/components/organisms/FidgetSettingsEditor.tsx (hunk 1 above)
Comment (sktbrd) [reply src/common/components/organisms/FidgetSettingsEditor.tsx]: the good news is that is in alphabetical order now ehehe

File: src/fidgets/community/nouns-dao/components/proposalListRowItem.tsx
Comment (sktbrd) [reply src/fidgets/community/nouns-dao/components/proposalListRowItem.tsx:1]: fixing ! thanks

**Comment by sktbrd** [comment]
@hiporox you can close this one, I am going to redo it from scratch , I deactivated the plugin that organizes the import, and the second time will be easier to go cleaner, thanks for the review and ur that , will submit a new cleaner one asap

**Pull request closed without merging**